tests/mock.py

```diff
@@ -2201,7 +2190,7 @@ def create_autospec(spec, spec_set=False, instance=False, _parent=None,
         else:
             parent = mock
             if isinstance(spec, FunctionTypes):
-                parent = mock.mock
```

Comment:
I'm... curious why this change happened.

Reply:
I looked at stdlib, it matches old version, so I reverted this change.

Reply:
Pretty sure that `mock.mock` is just flat out incorrect in all cases. The expected use of the mock module is that `mock/__init__.py` imports all objects from `mock/mock.py` and you use them at the top level.